### DISCUSSION TITLE
Need 3DFX testing CD ?

### DISCUSSION THREAD

**Discussion by stlintel** [bochs-emu/Bochs#177] [General]
I have the CD-Image with called 3dfxdreams.iso with a lot of stuff to play with on Bochs for 3DFX.
This is the contents of the CD:

![image](https://github.com/bochs-emu/Bochs/assets/77239907/36161f0f-16ae-47d2-8751-cb0eb8af0525)

Dozen of demos in Demos folder, More than 20 games samples, 2 benchmarks, dozen of 3DFX screensavers and a lot of other stuff.

Someone interested ? Vort ?

**Comment by sskras** [comment]
It would be nice to check if it's not on [archive.org](https://archive.org/) yet, and to upload it there in that case.
Eg: https://archive.org/details/3dfxVoodoo3

**Comment by Vort** [comment]
I found it here: https://web.archive.org/web/20170112041524/http://rottentreats.com/Files/3dfxDreams.rar

Also there are lots of content on this website: http://falconfly.3dfx.pl/3dfx.htm
I even found file named `3dfx_source_code.rar`.

It is more of a problem to properly process such amount of data, than to find it.
But thanks anyway, more data is better.
